examples/query-suggestions-history-with-category/app.tsx

```diff
@@ -0,0 +1,33 @@
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+import algoliasearch from 'algoliasearch';
+
+import '@algolia/autocomplete-theme-classic';
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  key: 'qs-with-rs-example',
```

Comment:
So that the recent searches are scoped to that example:

```suggestion
  key: 'query-suggestions-history-with-category-example',
```

Reply:
fixed.